language-design.md

```diff
@@ -38,7 +69,14 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
-
+A well-designed language is extendible, has a relatively small core vocabulary, is easy to learn (though not necessarily for everyone),
+is easy for the user to read and write, and has good documentation.
```

Comment:
Do you think good documentation is as important for novice programmers as for professional programmers? I know that when I was new to CS, good documentation would've done almost nothing for me, because finding and reading  documentation is really a (often overlooked) skill that you develop as you grow as a software developer.